### ISSUE TITLE
Feature Request: Add Trello Integration

### ISSUE THREAD

**Issue by petems** [Netflix/Scumblr#185]
After seeing this Tweet:

https://twitter.com/neilstudd/status/866254085439332352

Thought it might be an idea to add Trello search to Scumblr 😅

**Comment by sbehrens** [comment]
this may be accomplished using google search with the right dorks.

**Comment by petems** [comment]
Indeed, _but_ but it'd be nicer to catch this stuff before it gets indexed by Google! 😄

**Comment by sbehrens** [comment]
Glancing at the API a trello task could be added to monitor boards associated with a user (or org) but not globally.  This could be useful for folks wanting to watch their own org's boards.
